### ISSUE TITLE
Moving a file results in an empty file

### ISSUE THREAD

**Issue by aarongodin** [bchelli/node-smb2#26]
I've made what would be the most trivial example of the `rename` function. After running the example, the file is created but it has zero bytes of data.

The example:

```js
const SMB2 = require('smb2')
const { buildPath } = require('./jobs/util/smb-client')

const c = new SMB2({
  share: process.env.NETWORK_DRIVE_SHARE,
  domain: process.env.NETWORK_DRIVE_DOMAIN,
  username: process.env.NETWORK_DRIVE_USERNAME,
  password: process.env.NETWORK_DRIVE_PASSWORD,
  autoCloseTimeout: 0
})

const workFilePath = buildPath('CodeImport', 'Work', 'test')
const wipFilePath = buildPath('CodeImport', 'WIP', 'test')

c.rename(workFilePath, wipFilePath, (err) => {
  if (err) {
    console.error(err)
  }

  c.close()
})
```

**Comment by aarongodin** [comment]
This is fixed by #20.

**Comment by bchelli** [comment]
The version 0.2.8 now contains the fix by #20 
Thanks,
